torch_audiomentations/augmentations/colored_noise.py

```diff
@@ -0,0 +1,117 @@
+from math import ceil
+
+import torch
+
+from ..core.transforms_interface import BaseWaveformTransform
+from ..utils.dsp import calculate_rms
+from ..utils.io import Audio
+
+
+def _gen_noise(f_decay, num_samples, sample_rate):
+    """
+    Generate colored noise with f_decay decay using torch.fft
+    """
+    noise = torch.normal(0, 1, (sample_rate,))
```

Comment:
Could we ensure that this tensor is created on the same device as the input audio? That way we can avoid moving data from the CPU to the GPU

Something like this? https://stackoverflow.com/a/59179927/2319697

Reply:
Seems feasible enough, I will adjust accordingly.